SmplEditor/Song.cs

```diff
@@ -10,20 +10,54 @@ namespace SmplEditor
     public class Song
     {
         private string title;
+        public string Title{
+            get{
+                return this.title;
+            }
+        }
         private string artist;
+        public string Artist{
+            get{
+                return this.artist;
+            }
+        }
+        private SmplSong smplMusic = default(SmplSong);
+        public SmplSong SmplMusic {
+            set {
+                this.smplMusic = value;
+            }
+            get {
+                return this.smplMusic;
+            }
+        }
+        private ITunesLibraryParser.Track iTunesSong = default(ITunesLibraryParser.Track);
+        public ITunesLibraryParser.Track ITunesSong {
+            set {
+                this.iTunesSong = value;
+            }
+            get {
+                return this.iTunesSong;
+            }
+        }
+
         public Song(SmplSong smplMusic) {
             this.smplMusic = smplMusic;
             this.title = smplMusic.title;
             this.artist = smplMusic.artist;
         }
 
+        public Song(ITunesLibraryParser.Track iTunesMusic){
+            this.iTunesSong = iTunesMusic;
+            this.title = iTunesMusic.Name;
+            this.artist = iTunesMusic.Artist==default(string)?"unknown":iTunesMusic.Artist;
+        }
         public bool HasSmplSong() {
             return this.smplMusic != default(SmplSong);
         }
         public bool HasITunesSong() {
             return this.iTunesSong != default(ITunesLibraryParser.Track);
         }
-        // An iTunes song have
+        // An iTunes track have
```

Comment:
the naming "track" "song" and "music" should be sorted out.
Different wordings were used to avoid confusion with the type `Song`, but without any clear guideline.

Reply:
issue #8 posted.